### ISSUE TITLE
Get current modules floorplan easily

### ISSUE THREAD

**Issue by colinschmidt** [ucb-bar/hammer#368]
It would be nice to not have to parse through `get_placement_constraints()` to find the current modules width and height.

**Comment by colinschmidt** [comment]
See https://github.com/ucb-bar/hammer-cad-plugins/blob/a714e6c1e106f8b2fb48cdfc0acd970834d92ef2/par/innovus/__init__.py#L235 for an example use case

**Comment by edwardcwang** [comment]
Essentially a helper function that does `get_toplevel_placement_constraint`?

**Comment by colinschmidt** [comment]
Yeah something like that would be helpful. I also thought maybe you might want `get_placement_for_inst("MyInstance")` where no argument or something produces toplevel constriant. However I don't currently have a use case for that.
